src/deluge/model/instrument/midi_instrument.cpp

```diff
@@ -1121,3 +1164,29 @@ void MIDIInstrument::sendNoteToInternal(bool on, int32_t note, uint8_t velocity,
 		MIDITranspose::doTranspose(on, note);
 	}
 }
+
+// used with the renameMidiCCUI class to check if you're renaming a MIDI CC to the same name as
+// another midi CC
+int32_t MIDIInstrument::getCCFromName(String* name) {
+	for (int32_t i = 0; i < kNumRealCCNumbers; i++) {
+		if (ccNames[i].equalsCaseIrrespective(name)) {
+			return i;
+		}
+	}
+	return 255;
+}
+
+void MIDIInstrument::getNameFromCC(int32_t cc, String* name) {
```

Comment:
Since this can't error it would be clearer to return a string instead of taking a return pointer as an input